hera_qm/utils.py

```diff
@@ -610,3 +613,66 @@ def strip_extension(path, return_ext=False):
         return (root, ext[1:])
     else:
         return os.path.splitext(path)[0]
+
+
+def apply_yaml_freq_time_flags(uv, a_priori_flag_yaml, lat_lon_alt_degrees=None, telescope_name='HERA'):
```

Comment:
You may want to get the telescope name from the `uv` object, but then allow the user to overwrite it (so default None).

Reply:
changed default to None. If none, obtained from `uv`.